source/includes/_user_group.md

```diff
@@ -368,9 +368,10 @@ Property | Type | Description
 --------- | ------- | -----------
 `group_name` | string | The group name.
 `ldap_group_dn` | string | Optional parameter, The distinguishing name of the corresponding LDAP group, if appropriate.
+`role` | string | Optional parameter, The name of the system role for the group (default: "default").
```

Comment:
I don't know if it's clear enough.. it's a system role for the users that in the group, not a role for the group..